app/discord/draw.js

```diff
@@ -48,9 +48,16 @@ export default async function draw(message, opt) {
   }
 
   log.debug(`PGBG: ${message.author.username} drew the ${card.name} from ${opt} - ${card.image}`);
-    
   // build the message
-  const image = await images.draw(`./assets/decks/${opt.toLowerCase()}/${card.image}`, reversed ? 180 : 0);
+  var image;
+  if (card.file) {
+    image = await images.draw(
```

Comment:
nit: stylistically, I would have put this all on a single line

Reply:
which lines into one line?

Reply:
See the git diff on line `53` for the original, which has the entire assignment to `image =` on one line. This is purely personal preference, you can tell me to fuck off and merge it.

Reply:
i see. i think in this case it goes over an 80 character per line soft rule in my editor.